src/Services/FHIR/FhirPractitionerService.php

```diff
@@ -301,7 +271,7 @@ public function updateOpenEMRRecord($fhirResourceId, $updatedOpenEMRRecord)
      */
     public function searchForOpenEMRRecords($openEMRSearchParameters, $puuidBind = null)
```

Comment:
this seems odd that there is a parameter (`$puuidBind`) here that is not being used

Reply:
disregard above post. i see why it is there :)